crates/bevy_render/src/base_bundle.rs

```diff
@@ -0,0 +1,59 @@
+use bevy_ecs::prelude::Bundle;
+use bevy_transform::prelude::{GlobalTransform, Transform};
+
+use crate::view::{ComputedVisibility, Visibility};
+
+/// A [`Bundle`] with the following [`Component`](bevy_ecs::component::Component)s:
+/// * [`Visibility`] and [`ComputedVisibility`], which describe the visibility of an entity
+/// * [`Transform`] and [`GlobalTransform`], which describe the position of an entity
+///
+/// * To show or hide an entity, you should set its [`Visibility`].
+/// * To get the computed visibility of an entity, you should get its [`ComputedVisibility`].
+/// * To place or move an entity, you should set its [`Transform`].
+/// * To get the global position of an entity, you should get its [`GlobalTransform`].
+/// * For hierarchies to work correctly, you must have all four components.
+///   * You may use the [`BaseBundle`] to guarantee this.
+#[derive(Bundle, Debug, Default)]
+pub struct BaseBundle {
+    /// The visibility of the entity.
+    pub visibility: Visibility,
+    /// The computed visibility of the entity.
+    pub computed: ComputedVisibility,
+    /// The transform of the entity.
+    pub local: Transform,
+    /// The global transform of the entity.
+    pub global: GlobalTransform,
```

Comment:
I'm not keen on the local and global naming. Local and global what?
```suggestion
    /// The transform of the entity.
    pub transform: Transform,
    /// The global transform of the entity.
    pub global_transform: GlobalTransform,
```

Reply:
too much copy pasta 👍